src/Shell/CommandListShell.php

```diff
@@ -68,6 +69,11 @@ public function main()
             return;
         }
 
+        if (!empty($this->params['version'])) {
```

Comment:
If we had a ``param()`` method, we could make this more readable **and** get rid of the use of ``empty()``.